engines/sci/graphics/screen_item32.h

```diff
@@ -261,6 +261,13 @@ class ScreenItem {
 	void update(const reg_t object);
 
 	/**
+	 * Updates the properties of the screen item for one not belonging
+	 * to a VM object.
+	 */
+	void update();
```

Comment:
Please annotate this was originally UpdateScreenItem (and actually maybe we want to not ‘cheat’ on this quite yet, since this method is used also by kUpdateLine, kUpdatePolygon, and Robot? not sure. probably doesn’t matter)